cg/cli/backup.py

```diff
@@ -1,47 +1,46 @@
 import logging
+from typing import Optional
 
 import click
-
 from cg.apps.pdc import PdcApi
 from cg.meta.backup import BackupApi
-from cg.store import Store
+from cg.models.cg_config import CGConfig
+from cg.store import Store, models
 
 LOG = logging.getLogger(__name__)
 MAX_FLOWCELLS_ON_DISK = 1250  # Increased by 250 when the previous limit of 1000 was reached
 
 
 @click.group()
-@click.pass_context
-def backup(context: click.Context):
+def backup():
     """Backup utilities."""
     pass
 
 
 @backup.command("fetch-flowcell")
 @click.option("-f", "--flowcell", help="Retrieve a specific flowcell")
 @click.option("--dry-run", is_flag=True, help="Don't retrieve from PDC or set flowcell's status")
-@click.pass_context
-def fetch_flowcell(context: click.Context, dry_run: bool, flowcell: str):
+@click.pass_obj
+def fetch_flowcell(context: CGConfig, dry_run: bool, flowcell: str):
     """Fetch the first flowcell in the requested queue from backup."""
-    status_api = Store(context.obj["database"])
-    max_flowcells_on_disk = context.obj.get("max_flowcells", MAX_FLOWCELLS_ON_DISK)
-    root_dir = context.obj["backup"]["root"]
+    status_api: Store = context.status_db
+    max_flowcells_on_disk: int = context.max_flowcells or MAX_FLOWCELLS_ON_DISK
```

Comment:
was this not always None?

Reply:
I will leave it since it sounded on Barry like it should be a variable that is set via the config